app/src/main/java/org/oppia/android/app/devoptions/vieweventlogs/ViewEventLogsViewModel.kt

```diff
@@ -13,7 +13,7 @@ import javax.inject.Inject
  */
 @FragmentScope
 class ViewEventLogsViewModel @Inject constructor(
-  debugEventLogger: DebugEventLogger,
+  debugEventLogger: DebugAnalyticsEventLogger,
```

Comment:
Rename `debugEventLogger ` to `debugAnalyticsEventLogger`.

Same applies for all other files too.

Reply:
done